go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/xperimental/nextcloud-exporter
 go 1.17
```

Comment:
Can you update this as well with `go mod tidy -go 1.19`?

Reply:
Sure, done!

Reply:
Can you run `go mod tidy`, so that it updates the `go.sum` file as well?

Reply:
Whoops, missed that. Done!